api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/reshaping/RepeatVector.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.api.core.layer.reshaping
+
+import org.jetbrains.kotlinx.dl.api.core.KGraph
+import org.jetbrains.kotlinx.dl.api.core.layer.Layer
+import org.jetbrains.kotlinx.dl.api.core.shape.TensorShape
+import org.tensorflow.Operand
+import org.tensorflow.Shape
+import org.tensorflow.op.Ops
+import org.tensorflow.op.core.Tile
+
+/**
+ * Layer that repeats the input [n] times.
+ *
+ * Input shape: `2D tensor of shape (num_samples, features)`.
+ *
+ * Output shape: `3D tensor of shape (num_samples, n, features)`.
+ *
+ * @property n Repetition factor.
+ * @property [name] Custom layer name.
+ * @constructor Creates [RepeatVector] object.
+ *
+ * @author Stan van der Bend
+ * @since 0.2
+ */
+public class RepeatVector(
+    public val n: Int,
+    name: String = ""
+) : Layer(name) {
+
+    override fun build(tf: Ops, kGraph: KGraph, inputShape: Shape) {
+        //left empty
+    }
+
+    override fun computeOutputShape(inputShape: Shape): Shape {
+        require(inputShape.numDimensions() == 2) { "input tensor must have 2 dimensions" }
```

Comment:
I saw only the phrases starting from big letter as checks reports. Keep the convention to have a well documented library

Reply:
I am a bit confused on this point. I wasn't sure on how to handle this assertion, so I used something I saw used elsewhere in the project. Any suggested change?

Reply:
I was only thinking about a small typo here so just write 
```kotlin
require(inputShape.numDimensions() == 2) { "Input tensor must have 2 dimensions but got ${inputShape.numDimensions()}" }
```
as I understand one of KotlinDL goals i sto provide readable errors for user that can be easily fixed